### DISCUSSION TITLE
Execution Log: Column with RowCount

### DISCUSSION THREAD

**Discussion by GuenterSch** [dbeaver/dbeaver#17075] [General]
In the execution log, it would be nice, if there exist a column **Count** with the row count of a "select count(*) from" after the column Rows. This will be executed, if I press "Calcualte the total row count" in the status line.

![grafik](https://user-images.githubusercontent.com/99535516/178506870-2688c9f6-abbf-4ccb-b7a6-64015bdba4d8.png)

So it is possible to see in the log how many rows the previous select statement returned.

Thanks, BR
Günter

**Comment by E1izabeth** [comment]
Thank you for suggestion. We discussed it and assumed that it may lead to performance issues. The execution of such a query may take a while for big databases.
Besides that, if it's a statement that you executed from a script, you can click on the Re-execute button to see result.

![image](https://user-images.githubusercontent.com/28875055/178750603-f9fa2a27-5509-4278-b652-7de773a0deae.png)

But note: re-execute button doesn't work for embedded databases or if setting `Preferences->Editors->SQL Editor->Open separate connection for each editor` is disabled

**Reply by GuenterSch** [reply]
Hi Elizabeth,
my intent was not to execute the count(*) for every statement. Yes, this can take a very long time for big tables.

**My idea was: In case of an select count(*) - "Calcualte the total row count" - the row count will be shown/stored in a seperate column.**

The re-execute is not the same, because it can take a long time, and the row count may already have been changed.
I hope you can discuss this topic again, and implement the additional column.
Thanks, BR
Günter

**Reply by E1izabeth** [reply]
Thank you for reply and additional explanations.
